vault/resource_database_secret_backend_connection.go

```diff
@@ -881,6 +879,30 @@ func postgresConnectionStringResource() *schema.Resource {
 	return r
 }
 
+func mongodbConnectionStringResource() *schema.Resource {
+	r := connectionStringResource(&connectionStringConfig{
+		includeUserPass: true,
+	})
+	r.Schema["write_concern"] = &schema.Schema{
+		Type:        schema.TypeString,
+		Optional:    true,
+		Description: "Specifies the MongoDB write concern for Vault management operations.",
+	}
```

Comment:
If this is a JSON string, we should normalize it before writing to state. When users input JSON, they may format it differently (e.g., varying whitespace, different key ordering). If you don't normalize it, Terraform will see a difference between the input and the saved state and report constant drift on every plan.

See example here: https://github.com/hashicorp/terraform-provider-vault/blob/bcb1be4983c816359dbdf9956ccf384fefb62cd0/vault/resource_generic_endpoint.go#L49-L50